koncept/chapter1-5.tex

```diff
@@ -253,7 +253,7 @@ \subsubsection{Elektromagnetiska vågor}
 
 De magnetiska och elektriska fälten omkring en ledare är vinkelrätt orienterade
 mot varandra.
-Det elektromagnetiska fält, som de bildar tillsammans, bildar en vågfront som
+Det elektromagnetiska fält som de bildar tillsammans bildar en vågfront som
```

Comment:
*bildar* kommer två gånger. Låter det här bättre?

> Det elektromagnetiska fält som de bildar tillsammans **skapar** en vågfront som ...

Reply:
Hela förklaringsmodellen ger mig huvudvärk. Jag får återkomma till den tror jag. Det är någon sorts voodoo som smugit sig in.

Reply:
Ja, det är bättre att inte ha två "bildar" efter varandra.